detect.yml

```diff
@@ -1,14 +1,13 @@
 openapi: 3.0.3
-
-
 paths:
   /detect:
     parameters:
       - $ref: './common.yml#/components/parameters/x-request'
     post:
       tags:
         - matching
-      summary: "Detect face, evaluate attributes, assess the portrait quality"
+      summary: "detect faces"
```

Comment:
remove summary at all

Reply:
removed in https://github.com/regulaforensics/FaceSDK-web-openapi/pull/92/files